testing/py3-pillow-simd/APKBUILD

```diff
@@ -0,0 +1,38 @@
+# Maintainer: Marcel Esser <marcel.esser@gmail.com>
+pkgname=py3-pillow-simd
+_pkgname=Pillow-SIMD
+pkgver=6.0.0
+pkgrel=0
+pkgdesc="Python Imaging Library SIMD"
+url="https://github.com/uploadcare/pillow-simd"
+arch="x86_64"
+license="PSF"
+provides="py-pillow py3-pillow"
```

Comment:
I'd rather have people explicitly install this package

Reply:
Resolved in 2adbbd8fe9a12735d28542865a16663851bde0fd